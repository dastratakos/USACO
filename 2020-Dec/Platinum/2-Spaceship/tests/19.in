60 60 60
101101111111111011101110001110111111011000111100011110011110
111111011011010111101011110111111010111011110000111000111000
101011101110011111111100001101111111001101001001011011001011
111001111110101000010100111110000111101100110101110111110111
111101000101011010111000001011101111011100111100111111111010
110101110101111010101101011110100101011100111111000011111001
111111011011110111101011111111110011111001110111110101010111
100001111111101010001001111100110111011011111010111111110101
111111111111001111110111100011101110111101110011011010110111
001100100111101110101010111111101111000101111011100011100100
111111111001101111011101101010111110101011011101101011111110
001101110111001111111011010001111111111011011011111010011100
001011010111111110110001111101111001011001111111010111111101
011101010111111001111010101101110010011111110110111111111001
011111101111111101011011111100100110111111011010011111011010
101011111010111111111110011111010111110111111101111110011111
101110011111011110110101010100111110111111101001010111101111
100010010100101011101100101111111010111010111100111011111011
011110100111001011010111010101011110101110001101111111010101
110111100000111101111111111000000110111110101011001001111000
111111101100000001110111101011011101111001011101111010110111
010101010110001111110111111111011001011111010111101111110011
100000111010011011001111101000101110101111011001011111001001
100101110111110111111011011110100110011011000111101001011110
111110100110100111111100110111011111110101110110111101101001
111101100010111011011111111010110011111100110111111001110111
110001101111010101011111011011010000011101101110011100100110
001101110001101111000001110110111110111111111011011011001111
011011111101010110011111001101101111000110011001110000111110
111111110111101111111111111110010111111111101001101111011001
101010110110100001111111111101110100110000111111011111001100
010111110111111110001100101111110101110111111101101100101110
011011101001111000111110001000101111010011101111111011111101
011111111110100110110110110001011110111011011101101110101101
110110111010111010100011011111101111111110111001001011100011
100100001111111011010001110111111101111100111110111111101110
110011000110100001011110111000111001010100100111010101111110
110011111011111010111111101110110110101101110011111101111011
110100111011000011111011100011010110001111101110010101011110
101110111111110111111101111101110100100111110111111101101111
101001110101111110000100110110101111101111111110100100111100
011111010111111011101011110011111011111100110011111101110001
100010010111100101111011110011111101111111111111111111011011
011111101010011111111001001101001100101000100111011111110110
011110001001111111101011100110011100110101010010111101010011
111111111101110011111001100010101100001010111110111111101100
011011101011010101010000111010101101011000110011000111010111
101111010001110101111001101111111011111100111110111111111011
010110111110111111100100111111000011101101011011101111110011
100101011101100010011010111110101101111001111101110011111011
110101111111110011100100100111101111001010111011111111110110
101101111101110010011010101000001010001100100101011001010111
011111010111110100111010110100011111111010100101001110111110
101100111111010100101111110011100011110101110010101100111111
100111110010111110111110011110111101100100011111111010111111
101111101010100111111111100011110001111111101101010111000110
111101101111110101110111111100110111111111101111011010010111
001111111101001111011110111111100011001111110011111011010011
111001010110101101010101111010101101110111011111110110111111
101011001011011111001010110111111111110111110111111111110001
3 22 3 13
23 8 2 51
3 17 4 11
33 53 5 5
5 47 4 36
4 7 5 27
2 42 5 47
5 17 1 49
5 33 2 12
2 42 5 24
8 22 3 11
1 58 1 26
6 9 1 39
1 25 52 50
3 35 2 43
5 21 3 46
39 23 5 32
4 4 5 2
1 23 5 31
5 38 3 56
2 20 1 60
1 48 12 18
2 49 2 9
5 36 3 59
1 51 5 28
1 34 4 30
4 8 59 16
9 31 5 41
2 30 3 46
3 6 4 56
3 30 31 52
1 23 4 7
1 12 8 19
4 8 4 42
5 23 25 16
4 48 5 28
38 58 4 51
22 28 40 48
4 18 1 51
2 17 3 3
2 40 43 1
4 46 4 45
4 10 5 43
3 4 1 19
3 31 3 38
3 39 2 21
3 33 3 48
36 21 4 20
4 48 4 45
1 23 2 23
16 9 4 8
4 5 4 55
5 3 4 44
5 29 2 36
17 21 5 2
5 38 5 25
3 39 1 57
4 1 53 57
2 24 18 35
4 59 4 59
