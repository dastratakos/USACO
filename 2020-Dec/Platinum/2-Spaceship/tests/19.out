337199906
357369221
100399581
988710174
361612150
296168332
635450403
212252272
31942710
462436
725611091
826171804
20492828
420451233
205754
546986134
357403601
309834852
758966574
722517386
497368222
792578801
768973442
401063903
404364199
36433402
272780466
377128545
72449886
363802560
548930438
716869297
727674283
591115813
276130122
548331124
819496619
495400108
537497992
564863879
703364033
375172234
821821029
300254764
241392495
26068980
628479542
489222450
302980078
109734193
339891768
428345690
881159975
839552389
244557576
636111646
17363317
23032283
362817331
88452114
